mkdocs.yml

```
@@ -164,4 +164,4 @@ extra:
       icon: :material-cancel:{ .cancel }
       description: Unlikely to ever be supported or no upstream support.
 
-copyright: "Copyright &copy; 2014-2022"
+copyright: "Copyright &copy; 2014-2023"
```

Copyright years don't need to be updated every year, they're valid from the year the content was first created. https://hynek.me/til/copyright-years/. I think we should just remove the years entirely.

```suggestion
copyright: "Copyright &copy; Ibis Maintainers"
```